### DISCUSSION TITLE
Use spice instead of rdp for better mouse control

### DISCUSSION THREAD

**Discussion by KP2048** [casualsnek/cassowary#132] [Ideas]
When using MS Excel using cassowary, it is a chore to drag and resize charts because when i click to drag, it deselects almost immediately. I have to repeatedly click, drag a little bit, and click again to move/resize. as far as I know, this is an issue with most remote access software. if you could retrofit this to use the spice display stuff, this issue would go away, and it would even make playing games with this viable.

**Comment by 00sapo** [comment]
Agree, this would make the integration of quickemu easier. It could even lead to a single installation script!
